client/blocks/cookie-banner/index.tsx

```diff
@@ -0,0 +1,78 @@
+import { CookieBanner } from '@automattic/privacy-toolset';
+import cookie from 'cookie';
+import { useCallback, useEffect, useState } from 'react';
+import { useDispatch } from 'react-redux';
+import {
+	refreshCountryCodeCookieGdpr,
+	setTrackingPrefs,
+	shouldSeeCookieBanner,
+	getTrackingPrefs,
+} from 'calypso/lib/analytics/utils';
+import { bumpStat } from 'calypso/state/analytics/actions';
+import { useCookieBannerContent } from './use-cookie-banner-content';
+import type { CookieBannerProps } from '@automattic/privacy-toolset';
+
+export const isServer = typeof document === 'undefined';
+const noop = () => undefined;
+
+const CookieBannerInner = ( { onClose }: { onClose: () => void } ) => {
+	const content = useCookieBannerContent();
+	const dispatch = useDispatch();
+
+	const handleAccept = useCallback< CookieBannerProps[ 'onAccept' ] >(
+		( buckets ) => {
+			setTrackingPrefs( { ok: true, buckets } );
+			onClose();
+		},
+		[ onClose ]
+	);
+
+	useEffect( () => {
+		dispatch(
+			bumpStat(
+				'cookie-banner-view',
+				'total,' + document.location.host.replace( /[^a-zA-Z0-9]/g, '-' )
```

Comment:
Do we mean to add a single `-` for consecutive groups of replaced character, or one `-` per each replaced character?

```suggestion
				'total,' + document.location.host.replace( /[^a-zA-Z0-9]+/g, '-' )
```

Reply:
I moved this metric from the previous Gdpr Banner implementation 1:1 to avoid misalignments. When I try to see it in MC, the stats appear not to load - it looks broken, but I've never analyzed pixel stats.

Do you have some knowledge of that?

Reply:
Oh, that makes sense. I'm fine to keep it as-is then 👍 

It's broken for me as well, for the protocol. Might need to do some raw queries to double-check the data.

Reply:
I'll leave it unchanged for now, but I'll add fixing the stats for "cookie-banner-view" to our team's maintenance board.

Reply:
❤️ Thanks for that!